samples/drivers/watchdog/src/iwdg_main.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <watchdog.h>
+#include <misc/printk.h>
+
+/*
+ * The independent watchdog (IWDG) is clocked by its own dedicated
+ * 40kHz low-speed internal clock (LSI).
```

Comment:
LSI Clock frequency could have different values. For instance, it is 32KHz on STM32L4 series